### PR TITLE
Add ERC1484 external test

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/externalTests.sh [hunk 1]
```diff
@@ -89,3 +89,5 @@ test_truffle Zeppelin https://github.com/axic/openzeppelin-solidity.git solidity
 
 # Disabled temporarily as it needs to be updated to latest Truffle first.
 #test_truffle Gnosis https://github.com/axic/pm-contracts.git solidity-050
+
```
Comment (erak) [inline test/externalTests.sh:92]: Maybe a comment would be nice :)

**Comment by erak** [comment]
The failing external test does not seem to be related to the Node.js version. I've tried locally using ``v10.13.0``, but it still fails. I did not fully understand why, but they are using Websockets to communicate with Ganache: 
``networks: {
    development: {
      host: 'localhost',
      port: 8545,
      network_id: '*',
      websockets: true
},``
Could this be related? Do we need to tell Node to use WS?

**Comment by erak** [comment]
After thinking about this again, I'd vote for not adding this external dependency at all. It already introduced some maintenance overhead that we should not take care of. I think my standpoint is also pretty much in line with the feedback on our external tests that we got from Zeppelin.

**Comment by chriseth** [comment]
I agree. If it does not work out of the box then let's not add it.

**Pull request closed without merging**